### DISCUSSION TITLE
OIDC is not able to get cookie

### DISCUSSION THREAD

**Discussion by Abhikr1994** [OpenIDC/mod_auth_openidc#1053] [Q&A]
Hi. I'm attempting to use mod_auth_openidc but I'm seeing 'no state cookie found' errors. After login, the state cookie goes missing, and mod_auth_openidc redirects me back to the OIDC Deafult URL
Kindly check logs

![Screenshot from 2023-05-03 17-16-33](https://user-images.githubusercontent.com/21988446/235907413-56866a31-6ad8-4702-8ac4-d060bc40c45b.png)

![Screenshot from 2023-05-03 17-16-39](https://user-images.githubusercontent.com/21988446/235907435-800239d2-f1cb-494a-8c4a-8b366f9d476b.png)

Apache config
![Screenshot from 2023-05-03 17-19-25](https://user-images.githubusercontent.com/21988446/235907764-59ddc91c-d34d-4056-a45f-94321965187f.png)

![Screenshot from 2023-05-03 17-19-31](https://user-images.githubusercontent.com/21988446/235907782-70461b93-2df6-4679-b004-b9ba501ff088.png)

mod version - mod_auth_openidc-2.4.5

I am sending same grant and code from application to apache which it gets with authorization request.

**Comment by Abhikr1994** [comment]
I have set OIDCSameSiteCookie On but still its not working.
Hey either @zandbelt or someone else help  will be appreciated

**Reply by zandbelt** [reply]
probably a hostname mismatch between the URL you're trying to access and the Redirect URI, but it is hard to debug pictures...  https://meta.stackoverflow.com/questions/285551/why-should-i-not-upload-images-of-code-data-errors and you should upgrade to the latest version first
